frontend/src/routes/(app)/report/[owner]/[report]/+page.svelte

```diff
@@ -62,6 +62,23 @@
 	function handleMoved(e: CustomEvent) {
 		elements = e.detail.items;
 	}
+
+	function updateElementPosition(elementId: number | null | undefined, position: number) {
```

Comment:
Since this just swaps positions, maybe name it that?

Reply:
Was confused at first and might lead one to think they can move an element to a new position instead of swap the position with another element.

Reply:
SGTM!